diffraction/README.md

```diff
@@ -0,0 +1,18 @@
+# ess-legacy diffraction
+
+This folder contains Python scripts and Jupyter notebooks to reduce diffraction 
+data with scipp and scippneutron.
+
+The main script is `powder_reduction.py` with used event data from WISH at ISIS STFC.
+
+The required files i.e. sample, vanadium, empty instrument and calibration files are not provided.
```

Comment:
Could these files be shared publicly? If so, it might be nice to make them available.
We are also having to host more and more files for our neutron docs etc...
Maybe we should have a DRAM-wide discussion to try and find a solution for this?
Maybe like Mantid's FTP server for data files?